speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -187,27 +187,47 @@ Changed:
             racestopmenu.cpp
                 RmStopRaceMenu()
                     changed  AssistedHuman is a human
+                    removed controls menu
             legacymenu/
                 CMakeList.txt
                     added    ResearcherMenu.h and .cpp
                     added    DataSelectionMenu.h and .cpp
                     added    backend in both ADD_SDLIB_LIBRARY and ADD_SDLIB_INCLUDEDIR
-                mainscreens/mainmenu.cpp
-                    included "DataSelectionMenu.h"
-                    added    GoBack()
-                    changed  MainMenuInit()
+                    removed player, graph, opengl, advanced, simu, ai, hostsettings configs
+                mainscreens/
+                    mainmenu.cpp
+                        included "DataSelectionMenu.h"
+                        added    GoBack()
+                        changed  MainMenuInit()
+                        changed  onplayerconfigMenuActivate()
+                        changed  onRaceSelectActivate()
```

Comment:
onRaceSelectActivate() doesn't even exist and onRaceSelectMenuActivate() has not been changed.
MainMenuInit(), however, has been changed
```suggestion
                        changed  MainMenuInit()
```